tools/azure-pipelines/build-apache-repo.yml

```diff
@@ -70,7 +70,7 @@ stages:
             name: Default
           e2e_pool_definition:
             vmImage: 'ubuntu-20.04'
-          environment: PROFILE="-Dflink.hadoop.version=2.8.5 -Dinclude_hadoop_aws -Dscala-2.12"
+          environment: PROFILE="-Dinclude_hadoop_aws -Dscala-2.12 -Dflink.hadoop.version=3.1.3 -Phadoop3-tests,hive3"
```

Comment:
Why change it?

Reply:
> Why change it?

This modify is to run CI in Hadoop3 and Hive3 environment, and this commit will not to be merged.